charts/datahub/values.yaml

```diff
@@ -111,6 +111,11 @@ datahubUpgrade:
         cpu: 300m
         memory: 256Mi
 
+prometheus-kafka-exporter:
+  enabled: false
+  kafkaServer:
+  - prerequisites-kafka:9092  # <<release-name>>-kafka:9092
```

Comment:
We already have this value specified under `global.kafka.bootstrap.server`, can we not re-use that variable?

Reply:
I think it should be configured here: https://github.com/acryldata/datahub-helm/blob/master/charts/datahub/values.yaml

Reply:
we are using kafka-exporter directly from repository: https://prometheus-community.github.io/helm-charts  similar to schema-registry .So we need to define values under chart name